build.gradle

```diff
@@ -90,6 +97,10 @@ dependencies {
     // NOTE: Leave this at .11 as it must be an older version. if
     // you change it here you will also need to update JRubyExecIntegrationSpec
     integrationTestGems "org.jruby:jruby-complete:1.7.11"
+
+    gradleTest "org.jruby:jruby-complete:${jrubyVersion}"
+    gradleTest 'rubygems:credit_card_validator:1.1.0'
+    gradleTest 'org.bouncycastle:bcprov-jdk15on:1.50'
```

Comment:
I'm not sure I entirely understand or remember why this dependency should be here. I know we need it , but I thought we were putting it into the depepndency tree when we applied the base plugin?